mesheryctl/internal/cli/root/system/system.go

```diff
@@ -27,7 +27,9 @@ import (
 )
 
 const (
-	fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	fileURL         = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	manifestsURL    = "https://api.github.com/repos/layer5io/meshery/git/trees/3ba314a0870d5291be6216b4d60d2bc9675a39b2"
+	rawManifestsURL = "https://raw.githubusercontent.com/layer5io/meshery/master/install/deployment_yamls/k8s/"
```

Comment:
always download from master, or should it match the version of `mesheryctl`? maybe parametrize `master`.
should we package and version these files, eventually (would surely be for another issue, I assume)

Reply:
Yes, it should actually check the context and get the files according to `channel` and `version`. I felt like that would be a bigger change and thought I would add it after this in a separate PR.

Reply:
:+1: agreed